src/badfish/badfish.py

```diff
@@ -1999,8 +1999,11 @@ async def make_gif(self, minutes, interval):
         frames = []
         files = []
         size = (720, 400)
-        for _ in range(iterations):
-            filename = await self.get_screenshot(gif=True)
+        for i in range(iterations):
+            if i == 0:
+                filename = await self.get_screenshot(gif=True, first=True)
+            else:
+                filename = await self.get_screenshot(gif=True)
```

Comment:
You can have this if/else in a one-liner like this:
filename = await self.get_screenshot(gif=True, first=i==0)

Reply:
Yeah, that's probably a nicer solution 😅